plugins/amazonq/mynah-ui/src/mynah-ui/ui/quickActions/generator.ts

```diff
@@ -78,11 +77,6 @@ export class QuickActionGenerator {
                             },
                         ]
                         : []),
-                ],
-            },
-            {
-                groupName: `Q Developer Agent for <b>Code Transformation</b>`,
```

Comment:
Should this stay?

Reply:
AFAIK this was removed from the VSC impl